code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -380,6 +382,14 @@
 
 	return dat
 
+/mob/living/silicon/robot/verb/toggle_panel_lock()
+	set name = "Toggle Panel Lock"
+	set category = "Silicon Commands"
+	to_chat(src, "You begin [locked ? "" : "un"]locking your panel.")
+	if(!opened && has_power && do_after(usr, 80) && !opened && has_power)
```

Comment:
why does this need a doafter any reason it cant be instant?

Reply:
In case someone prying off panel